bin/registerServerTopology.php

```diff
@@ -137,7 +137,7 @@
         $configOptions['TARGET_NODE_ADDRESS'] . ': please enter your password ',
         true
     );
-    $configOptions['PROXY_USAGE'] =  strtolower(askQuestion("Are you using a proxy ? (y/n)"));
+    $configOptions['PROXY_USAGE'] =  strtolower(askQuestion("Are you using a proxy ? (y/n) "));
```

Comment:
In English we shouldn't use space before ponctuation
```suggestion
    $configOptions['PROXY_USAGE'] =  strtolower(askQuestion("Are you using a proxy? (y/n) "));
```